src/pages/race.astro

```diff
@@ -0,0 +1,153 @@
+---
+import Layout from "../layouts/Layout.astro";
+import JoinCards from "../components/JoinCards.astro";
+
+
+import { Image } from "astro:assets";
+import teamImage from "../data/images/team_4.jpg";
+import bunny from "../data/images/bunny.png";
+import carimage from "../data/images/carmolecule.png";
+import typesofcars from "../data/images/typesofcars.png";
+---
+
+</main>
+<Layout title="The Next Race">
+    <main>
+      <Image
+        src={bunny}
+        alt="Formula Nano Team Members on a sunny day standing in front of University of Waterloo Quantum Nano Center"
+        class="mt-8 h-48 w-full rounded-2xl object-cover sm:h-[30rem] object-top"
+      />
+      <h1 class="text-center py-8 text-5xl font-bold">Our Race to the Next Nanocar Competition!</h1>
```

Comment:
This should match the headings on other pages (larger text size and left aligned)